config/flownode.example.toml

```diff
@@ -0,0 +1,93 @@
+## The running mode of the flownode. It can be `standalone` or `distributed`.
```

Comment:
You can put the toml in `config-docs-template.md` and executes the `make config-docs`.